### PR TITLE
graph-push-hook: allow for node removal by author

### PR DESCRIPTION
**Pull request by liam-fitzgerald**

### CODE REVIEW

**Comment by matildepark** [comment]
This should probably include FE changes in the PR?

**Comment by liam-fitzgerald** [comment]
ruh roh

**Comment by matildepark** [comment]
Is that prohibitive? We just want to avoid a situation where back-end and front-end changes are deviated and it's unclear which support which.

**Comment by liam-fitzgerald** [comment]
no, I just forgot accidentally

File: pkg/arvo/app/graph-push-hook.hoon [hunk 1]
```diff
@@ -32,6 +32,22 @@
...
       (is-admin:grp src.bowl i.group-paths)
   ==
+::
+++  is-allowed-remove
+  |=  [=resource:res indices=(set index:store) =bowl:gall]
+  ^-  ?
+  =/  gra   ~(. graph bowl)
+  ?.  (is-allowed resource bowl %.n)
+    %.n
+  %+  levy
```
Comment (tacryt-socryp) [inline pkg/arvo/app/graph-push-hook.hoon:42]: What does this do? The method description in the standard library is... not helpful.

Comment (tacryt-socryp) [reply pkg/arvo/app/graph-push-hook.hoon:42]: Oh, okay. It just returns yes if all of the result tests are yes... essentially an ?& across a list.

File: pkg/arvo/lib/graph.hoon [hunk 2]
```diff
@@ -17,6 +17,18 @@
   %+  scry-for  update:store
   /graph/(scot %p entity.res)/[name.res]
 ::
+++  get-node
```
Comment (tacryt-socryp) [inline pkg/arvo/lib/graph.hoon:20]: Technically this is "got-node", as "get-node" would return a unit.

**Review by tacryt-socryp — CHANGES_REQUESTED** [review]
One small change (get to got), otherwise looks good.

**Review by tacryt-socryp — APPROVED** [review]

**Pull request merged**